src/js/CreateAnalysis/Components/SelectedFileListPanel.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Typography from "@material-ui/core/Typography";
+import Divider from "@material-ui/core/Divider";
+
+import SelectedFileList from "./SelectedFileList";
+
+import {inputConfig} from "../../../resources/config.js";
+const containerStyles = {
+  flexDirection: "row",
+  height: "50%",
+  backgroundColor: "#fdfdfd"
+};
+
+const SelectedFileListPanel = props => {
```

Comment:
```suggestion
const SelectedFileListPanel = ({ filePaths, onDelete }) => {
```

Reply:
I see this in SelectedFileList as well.. will change!